pkg/webhook/resources/setting/validator.go

```diff
@@ -665,3 +674,31 @@ func (v *settingValidator) checkStorageNetworkRangeValid(config *storagenetworkc
 	}
 	return nil
 }
+
+func (v *settingValidator) validateUpdateRancherManagerSupport(oldSetting *v1beta1.Setting, newSetting *v1beta1.Setting) error {
+	if newSetting.Value == "" {
+		return nil
+	}
+
+	value := strings.ToLower(newSetting.Value)
+	if (value != "true") && (value != "false") {
+		return werror.NewInvalidError("value should be true or false", "value")
+	}
+
+	if value == "false" {
+		return nil
+	}
```

Comment:
the `value` variable can be defined as a boolean type which will make it more readable and also help to simplify the code. e.g.,
```
enableManager, err := strconv.ParseBool(strings.ToLower(newSetting.Value))
if err != nil {
    return werror.NewInvalidError("value should be either true or false", "value")
}

if !enableManager {
    return nil
}
```

Reply:
Updated it. Thanks.